nodeJS/APIs/APIs.md

```diff
@@ -12,41 +12,42 @@ By the end of this lesson, you should be able to do the following:
 - Explain CORS headers
 
 ### Introduction
-In recent years, a new pattern for developing websites has been gaining popularity. Instead of creating an app that hosts both the database and view templates, many developers are separating these concerns into separate projects, hosting their database on a server (either on something like [Heroku](https://www.heroku.com/) or on a VPS like [Digital Ocean](https://www.digitalocean.com/)), then using a service such as [GitHub Pages](https://pages.github.com/) or [Netlify](https://www.netlify.com/) to host their frontend.
+In recent years, a new pattern for developing websites has been gaining popularity. Instead of creating an app that hosts both the database and view templates, many developers are separating these concerns into separate projects, hosting their backend and database on a server (either on something like [Heroku](https://www.heroku.com/) or on a VPS like [Digital Ocean](https://www.digitalocean.com/)), then using a service such as [GitHub Pages](https://pages.github.com/) or [Netlify](https://www.netlify.com/) to host their frontend. This technique is sometimes referred to as the [Jamstack](https://jamstack.org/what-is-jamstack/).
```

Comment:
>backend and database on a server

Not sure how to best word this, but I think it can be misleading to mentally correlate the server as only a database.

>This technique is sometimes referred to as the [Jamstack](https://jamstack.org/what-is-jamstack/).

I don't love that Netlify co-opted this into Jamstack but I think it's helpful to have a searchable keyword for this pattern